k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -20,9 +20,9 @@ actions:
       }
 
       function wait_for_healthy_sts {
-        info "Waiting for equal desired and current number of replicas"
+        stderr_msg "Waiting for equal desired and current number of replicas"
```

Comment:
Does this message display somewhere? It seems to me like `testrunner` doesn't know how to do it for now.

Reply:
Sure, make sens, will remove this.

We do not want to keep logging information in tests if this is not yet supported by testrunner.